framework/drupal/docker-compose.ide.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+# Adds Cloud9 IDE.
+services:
+  ide:
+    container_name: ddev-${DDEV_SITENAME}-ide
+    image: briangilbert/cloud9-alpine:20180318
```

Comment:
Is there a way we could use the upstream cloud9? This one is likely to be unmaintained, and the tag is already old obviously.

Also, I'm not sure how big this is, but would love to have running this be optional. Just have a script that runs it with the mount, rather than attaching it to ddev?